helpers/Cloudinary/handleDeleteImage.js

```diff
@@ -0,0 +1,12 @@
+export default async function handleDeleteImage(publicId) {
+  const responseDelete = await fetch("/api/upload", {
+    method: "DELETE",
+    body: publicId,
+  });
```

Comment:
sending a body as part of a DELETE request is not really a common practice.

You would normally do it over the API Route:

```suggestion
  const responseDelete = await fetch(`/api/upload/${publicId}`, {
    method: "DELETE",
  });
```

This would also mean creating a corresponding API Route.